### PR TITLE
Subtle message fix

### PR DESCRIPTION
**Pull request by KIBORG04**
## About the PR 
"{message}" not a interpolated variable

https://github.com/space-wizards/space-station-14/pull/12504

**Screenshots**

**Changelog**

:cl:
- fix: Send subtle message works correctly.

### CODE REVIEW

File: Content.Server/Prayer/PrayerSystem.cs [hunk 1]
```diff
@@ -69,7 +69,7 @@ public void SendSubtleMessage(IPlayerSession target, string messageString, strin
             return;
 
         _popupSystem.PopupEntity(popupMessage, target.AttachedEntity.Value, Filter.Empty().AddPlayer(target), PopupType.Large);
-        _chatManager.ChatMessageToOne(ChatChannel.Local, messageString, popupMessage + " \"{message}\"", EntityUid.Invalid, false, target.ConnectedClient);
+        _chatManager.ChatMessageToOne(ChatChannel.Local, messageString, popupMessage + $" \"{messageString}\"", EntityUid.Invalid, false, target.ConnectedClient);
```
Comment (KIBORG04) [inline Content.Server/Prayer/PrayerSystem.cs:72]: Nice meme, popupMessage also means a NAME of the voice in the head

Comment (Just-a-Unity-Dev) [reply Content.Server/Prayer/PrayerSystem.cs:72]: isn't it supposed to be $message since chat does some localization stuff

**Comment by moonheart08** [comment]
This is a webedit. Test your changes.

**Comment by Just-a-Unity-Dev** [comment]
i'll get a PR up soon (like 4 hours once i escape hell)

**Comment by KIBORG04** [comment]
This is not a webedit.

File: Content.Server/Prayer/PrayerSystem.cs (hunk 1 above)
Comment (KIBORG04) [reply Content.Server/Prayer/PrayerSystem.cs:72]: I was looking in the game and saw literally "god: {message}"

**Comment by KIBORG04** [comment]
For maintainers
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated

**Pull request closed without merging**